drivers/volume/portworx/portworx.go

```diff
@@ -4753,3 +4771,50 @@ func (d *portworx) UpdatePoolLabels(n node.Node, poolID string, labels map[strin
 	return nil
 
 }
+
+// GetPoolIOPriorityStatus returns IO Priority of the Pool
+func (d *portworx) GetPoolLabelValue(poolUUID string, label string) (string, error) {
```

Comment:
Fix function comment, its misleading

Reply:
Thanks for pointing this out ! I missed this earlier